packages/expo-passkeys/package.json

```diff
@@ -32,8 +32,10 @@
   },
   "dependencies": {
     "@clerk/shared": "workspace:^",
-    "@clerk/types": "workspace:^",
-    "expo-modules-core": "1.12.26"
+    "@clerk/types": "workspace:^"
+  },
+  "devDependencies": {
+    "expo": "~52.0.0"
   },
   "peerDependencies": {
     "expo": "*",
```

Comment:
Open for discussion regarding min version. `requireNativeModule` was only available since Expo SDK 49.

I think SDK 50 is good `^50 || ^51 || ^52`